package/com.unity.formats.usd/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "dependencies": {
+        "com.unity.timeline" : "1.0.0"
+    },
+    "description": "The USD package provides an API to manipulate Universal Scene Description (.usd) files. USD is a 3D Scene format that scales well to large scenes and contains assets of different types (geometry, images, etc).", 
```

Comment:
For approved description / tags : 
see : https://docs.google.com/document/d/1jhRFrI3PRp4GvgpVjl2Sa7CxC06jFhX2WuPKwfA-HV4/edit#